apps/site/src/app/(home)/sections/About/About.tsx

```diff
@@ -0,0 +1,67 @@
+import Image from "next/image";
+import Lantern from "@/lib/components/Lantern/Lantern";
+import koiJump from "@/assets/images/koi-jump.png";
+import styles from "./About.module.css";
+
+const About = () => {
+	return (
+		<>
+			<section className="container md mx-auto text-center my-14 w-3/4 sm:w-1/2 xl:w-2/5 relative">
```

Comment:
Thought: personally not a fan of this proportion based container sizing: the base 3/4 seems to not provide enough space on narrower views. There may be some bias with being used to the bucketed fixed widths we previously used with Bootstrap.

Reply:
I can switch over to the fixed width classes then.

Reply:
While Tailwind has [classes](https://tailwindcss.com/docs/width) that are fixed width, I don't see any that are large enough for what we want. I did replace `w-3/4` with `w-96` though.